pkg/controllers/provisioningv2/rke2/provisioninglog/provisioninglog.go

```diff
@@ -44,17 +45,19 @@ type handler struct {
 }
 
 func (h *handler) OnConfigMap(key string, cm *corev1.ConfigMap) (*corev1.ConfigMap, error) {
-	if cm == nil {
+	if cm == nil || !cm.DeletionTimestamp.IsZero() {
```

Comment:
Nit: I'd prefer `configMap` instead of `cm` I think it reads better

Reply:
There are a few places in Rancher where the same abbreviation is used already, plus it's already existing code and renaming `cm` in func signature would result in a lot of changes making reviewing it harder (unless put in a separate commit).